StandardConfig/production/ILDReconstruction.py

```diff
@@ -0,0 +1,346 @@
+import os
+import sys
+
+from Gaudi.Configuration import INFO
+
+from Configurables import (
+    LcioEvent,
+    MarlinProcessorWrapper,
+    k4DataSvc,
+    PodioInput,
+    PodioOutput,
+    EDM4hep2LcioTool,
+    Lcio2EDM4hepTool,
+    GeoSvc,
+    ApplicationMgr,
+)
+from k4FWCore.utils import import_from, SequenceLoader
+from k4FWCore.parseArgs import parser
+from k4MarlinWrapper.parseConstants import parseConstants
+
+DETECTOR_MODELS = (
+    "ILD_l2_v02",
+    "ILD_l4_o1_v02",
+    "ILD_l4_o2_v02",
+    "ILD_l5_o1_v02",
+    "ILD_l5_o1_v03",
+    "ILD_l5_o1_v04",
+    "ILD_l5_o1_v05",
+    "ILD_l5_o1_v06",
+    "ILD_l5_o2_v02",
+    "ILD_l5_o3_v02",
+    "ILD_l5_o4_v02",
+    "ILD_s2_v02",
+    "ILD_s4_o1_v02",
+    "ILD_s4_o2_v02",
+    "ILD_s5_o1_v02",
+    "ILD_s5_o1_v03",
+    "ILD_s5_o1_v04",
+    "ILD_s5_o1_v05",
+    "ILD_s5_o1_v06",
+    "ILD_s5_o2_v02",
+    "ILD_s5_o3_v02",
+    "ILD_s5_o4_v02",
+)
+
+parser.add_argument(
+    "--inputFiles",
+    action="extend",
+    nargs="+",
+    metavar=["file1", "file2"],
+    help="One or multiple input files",
+)
+parser.add_argument(
+    "--compactFile", help="Compact detector file to use", type=str, default=""
+)
+parser.add_argument(
+    "--outputFileBase",
+    help="Base name of all the produced output files",
+    default="StandardReco",
+)
+parser.add_argument(
+    "--lcioOutput",
+    help="Choose whether to still create LCIO output (off by default)",
+    choices=["off", "on", "only"],
+    default="off",
+    type=str,
+)
+parser.add_argument(
+    "--cmsEnergy",
+    help="The center-of-mass energy to assume for reconstruction in GeV",
+    choices=(250, 350, 500, 1000),
+    type=int,
+    default=250,
+)
+parser.add_argument(
+    "--detectorModel",
+    help="Which detector model to run reconstruction for",
+    choices=DETECTOR_MODELS,
+    type=str,
+    default="ILD_l5_o1_v02",
+)
+parser.add_argument(
+    "--perfectPFA", help="Run perfect PandoraPFA", action="store_true", default=False
+)
+parser.add_argument(
+    "--runOverlay",
+    help="Run background overaly. NOTE: You have to make sure that the Overlay algorithms in "
+    " BgOverlay/BgOverlay.py are provided with the necessary overlay files",
+    action="store_true",
+    default=False,
+)
+# BeamCal reco configuration
+parser.add_argument(
+    "--runBeamCalReco",
+    help="Run the BeamCal reco",
+    action="store_true",
+    dest="runBeamCalReco",
+)
+parser.add_argument(
+    "--noBeamCalReco",
+    help="Don't run the BeamCal reco",
+    action="store_true",
+    dest="runBeamCalReco",
+)
+parser.set_defaults(runBeamCalReco=True)
```

Comment:
```suggestion
# BeamCal reco configuration
parser.add_argument(
    "--runBeamCalReco",
    help="Run the BeamCal reco",
    action="store_true",
    dest="runBeamCalReco",
)
parser.add_argument(
    "--noBeamCalReco",
    help="Don't run the BeamCal reco",
    action="store_false",
    dest="runBeamCalReco",
)
parser.set_defaults(runBeamCalReco=True)
```
or am I missing something?

Reply:
No. I was just making copy-paste mistakes ;) Thanks for having another look.